src/main/java/io/github/townyadvanced/townycombat/utils/TownyCombatMovementUtil.java

```diff
@@ -174,15 +174,16 @@ public static void punishEncumberedJumpingPlayers() {
         final double JUMP_DAMAGE_THRESHOLD = TownyCombatSettings.getJumpDamageThreshold();
         final double JUMP_DAMAGE_PER_ENCUMBRANCE_PERCENT = TownyCombatSettings.getJumpDamageDamagePerEncumbrancePercent();
         Double playerEncumbrancePercentage;
-        double velocityY;        
-        double damage;
+        double velocityY;
         double newHealth;        
         for(Player player: Bukkit.getOnlinePlayers()) {
             if(player.getGameMode() == GameMode.CREATIVE
                     || player.getGameMode() == GameMode.ADVENTURE
                     || player.isInvulnerable()) {                
                 continue;
             }
+            if(!player.isOnGround)
```

Comment:
Missing a (), and the #isOnGround() is apparently unreliable (I still use it in TownyFlight though,) and is entirely client-side.

Reply:
Ok thanks. 

I  see that method is even marked depreciated. Swapping it out now for something more robust, to check for air underneath the player.

Reply:
Keep an eye on it, you may get a bunch of false positives, especially if they're in the area of a jump-altering beacon.